core/src/main/java/cucumber/runtime/RuntimeOptions.java

```diff
@@ -186,6 +189,10 @@ private void parse(List<String> args) {
                     addLineFilters(parsedLineFilters, key, pathWithLines.lines);
                 }
             }
+            if (strict && wip) {
+                System.err.println("You can't use both --strict and --wip");
```

Comment:
Why not?

Reply:
It is based on [Cucumber-Ruby](https://github.com/cucumber/cucumber-ruby/blob/debe5d3b419774cd3514222b4500f6bedca61611/lib/cucumber/cli/configuration.rb#L30)

Reply:
Ok, I do that.